build.xml

```diff
@@ -49,30 +50,45 @@
                 
     <!-- test-compile -->
     <target name="test-compile" depends="resolve-classpath">
-        <javac includeantruntime="false" srcdir="${test.dir}" destdir="${class.dir}"
+        <javac includeantruntime="false" srcdir="${test.dir}" destdir="${class.dir}/test"
             debug="true" debuglevel="lines,vars,source" fork="true">
             <compilerarg line="-J-Xmx4g" />
             <classpath refid="lib.classpath" />
             <classpath>
+                <pathelement location="${class.dir}" />
                 <pathelement location="${testcfg.dir}" />
             </classpath>
         </javac>
-        <copy todir="${class.dir}">
+        <copy todir="${class.dir}/test">
             <fileset dir="${test.dir}">
                 <include name="**/*.txt" />
                 <include name="**/*.tokens" />
                 <include name="**/*.list" />
                 <include name="**/*.xls*" />
+                <include name="**/*.ods" />
             </fileset>
         </copy>
+        <!-- Build the Windows-style test-expected-results files from the Mac/Linux ones -->
+        <copy todir="${class.dir}/test/resources/">
+            <fileset dir="${test.dir}/resources/">
+                <include name="**/*.err" />
+                <include name="**/*.out" />
+            </fileset>
+            <globmapper from="*" to="win_*" />
+            <filterchain>
+                <fixcrlf eol="crlf" srcdir="${class.dir}/test/resources/" destdir="${class.dir}/test/resources/" includes="win_*" fixlast="false" />
+                <replacestring from="test/resources/" to="test\resources\" />
+            </filterchain>
+        </copy>
     </target>
-    
+
     <!-- tests go here -->
     <target name="test" depends="compile, test-compile">
-        <java classname="${test.mainclass}" fork="true" failonerror="true">
+        <java classname="${test.mainclass}" fork="true" failonerror="true" dir="${class.dir}">
             <jvmarg value="-Xmx4g" />
             <classpath refid="lib.classpath" />
             <classpath>
+				<pathelement location="${class.dir}/test" />
```

Comment:
indentation needs fixing

Reply:
Will fix.

Reply:
Fixed in the open pull request from my `udiff` branch.